### PR TITLE
Docs update for deployment yaml reference

### PR DESCRIPTION
**Pull request by alexmasi**
- add docs on arista controller

### CODE REVIEW

File: docs/create_topology.md [hunk 1]
```diff
@@ -78,28 +189,35 @@ Keysight controller yaml:
...
 > because the SR Linux controller + containers require the `bridge` cluster CNI
 > instead of the default `ptp` cluster CNI that `kind` uses.
 
-To manually apply the controller run the following command:
+To manually apply the controller, first clone the `srl-controller`
+[repo](setup.md#srl-controller) and then run the following command:
 
 ```bash
-kubectl apply -k https://github.com/srl-labs/srl-controller/config/default
+kubectl apply -k srl-controller/config/default
```
Comment (hellt) [inline docs/create_topology.md:202]: Hi @alexmasi 
you don't need to clone the repo to install srl-controller.
Unless there was some hidden meaning you can say 

> To manually apply the controller run `kubectl apply -k https://github.com/srl-labs/srl-controller/config/default`

Comment (alexmasi) [reply docs/create_topology.md]: Thanks, I accidentally changed this from our internal docs differing instructions. Reverted

**Review by guoshiuan — APPROVED** [review]

**Pull request merged**